crates/goose-cli/src/session.rs

```diff
@@ -25,6 +25,32 @@ pub fn ensure_session_dir() -> Result<PathBuf> {
     Ok(config_dir)
 }
 
+pub fn get_most_recent_session() -> Result<Option<PathBuf>> {
```

Comment:
meganit: i find Result<Option<>> a bit weird here, might just do Option<PathBuf> and use https://doc.rust-lang.org/stable/std/result/enum.Result.html#method.ok to convert anything that would be an err?